ruby/initialize.js

```diff
@@ -253,8 +276,15 @@
             })
         },
         getDom(results, domId) {
-            return addCommand(ruby`get_dom(${results}, ${domId})`).ref('dom')
-        }
+            return addCommand(ruby`get_dom(${results}, ${domId})`).methods({
+                getNodesByAttribute: (dom, name) => addExpression(ruby`get_nodes_by_attribute(${dom}, ${name})`).type('Array')
+            }).ref('dom')
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/sdk.coverage.tests/pullRequest?prid=8528488)
```suggestion
            }).ref("dom")
```